docs/provision_hypershift_clusters_by_mce.md

```diff
@@ -129,31 +129,202 @@ Upon scaling up a NodePool, a Machine will be created, and the CAPI provider wil
 
 Upon scaling down a NodePool, Agents will be unbound from the corresponding cluster. However, you must boot them with the Discovery Image once again before reusing them.
 
-To use the Agent platform, the Infrastructure Operator must first be installed. Please see [here](https://hypershift-docs.netlify.app/how-to/agent/create-agent-cluster/) for details.
+To use the Agent platform, the Infrastructure Operator must first be installed. Please see [here](https://hypershift-docs.netlify.app/how-to/agent/create-agent-cluster/) for details or you can enable it through the multiclusterengine resource.
```

Comment:
We should mention we only officially support AI on `local-host`, using the ACM/MCE hub as the hosting service cluster.